pkgs/by-name/meta-press/package.nix

```
@@ -6,12 +6,12 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "meta-press";
-  version = "1.8.17.1";
+  version = "1.8.17.4";
 
   src = fetchgit {
     url = "https://framagit.org/Siltaar/meta-press-ext.git";
     rev = "v${finalAttrs.version}";
```

This was introduced a while ago to avoid conflicts with branches that share the same name as the tag. Although this package's source repo doesn't have such branches, IMO it's a good pattern to adopt when working with tags.

```suggestion
    tag = "v${finalAttrs.version}";
```

---

You can do `git commit --amend` and it will include the changes to the latest commit. Then you can force-push the branch. Alternatively we can rebase, but amending is simpler.

---

I just did that and it worked. Thank you for the review @eljamm ❤️